tests/integration/test_client.py

```diff
@@ -57,3 +57,14 @@ async def test_client_start_transcribe_stream_bad_request(self, client):
                 media_sample_rate_hz=9999999,
                 media_encoding="pcm",
             )
+
+    @pytest.mark.asyncio
+    async def test_client_start_transcribe_stream_bad_header(self, client):
```

Comment:
Can we get some tests with values other than `true`? What happens when we get `true` and `false`, or `true` and an unknown value? What happens when the end user miscases their input?